quality/jacoco/android.gradle

```diff
@@ -13,7 +13,7 @@ tasks.check.dependsOn tasks.jacoco
 
 project.afterEvaluate {
   def variants = android.hasProperty('libraryVariants') ? android.libraryVariants : android.applicationVariants
-  println variants 
+  println variants
```

Comment:
think we should remove this line (i think i left it when i was debugging, my bad)